### DISCUSSION TITLE
[Wiki] Multi webui configuration for nginx is broken because of default mime-types

### DISCUSSION THREAD

**Discussion by namniav** [CzBiX/qb-web#149] [Show and tell]
When using the example from https://github.com/CzBiX/qb-web/wiki/Running-multi-WebUI-at-the-same-time#nginx, the Web UI is broken because all script files are served with header `Content-Type: text/plain`. 
Add `include mime.types;` to let nginx auto-detects mime-types.
```
  location / {
    include mime.types;
    root /var/www/qb-web/public/;
  }
```

**Comment by CzBiX** [comment]
The following configuration should be included in the default configuration of nginx. Is it because you removed the default configuration that you need to add it here again?
`include       /etc/nginx/mime.types;`

**Reply by namniav** [reply]
Yes, just realized the default configuration is overwriten by docker's volume mounting. I'm using a fresh new configuration.
